src/common/comment.ts

```diff
@@ -114,4 +122,5 @@ export interface Comment {
 	isDraft?: boolean;
 	inReplyToId?: number;
 	graphNodeId: string;
+	reactions?: Reaction[];
```

Comment:
I think we should also add this for the REST API case, but can do that separately